demo/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Adding new comment to continue conversation
```
Shiva: @khushal87 minor feedback
also I think we should update packge.json for the demo folder as well

Khushal:
With this?

"build": "tsc",
@sshivananda
```

Reply:
yeah - I think the build script would be useful 
apart from that I think we also need three dev dependencies
```
"@types/node": "^14.14.37",
 "@types/react-native": "~0.62.13",
"typescript": "^4.2.3"
```